version_test.go

```diff
@@ -1,14 +1,14 @@
-// +build unit
-
 package main
 
 import (
 	"testing"
 
 	"github.com/almighty/almighty-core/app/test"
+	skipper "github.com/almighty/almighty-core/test"
 )
 
 func TestAuthorizeLoginOK(t *testing.T) {
+	skipper.SkiptTestIfNotUnitTest(t)
```

Comment:
I like the latter idea. But the first idea, admittedly it reads much better, doesn't say what it does. In other words: `Integration` and `Unit` are not verbs not sentences. I think I'll go with the last one. But How do we make sure that there is a Database or Mail for example?